build.gradle

```
@@ -99,6 +160,7 @@ dependencies {
     api("org.opensearch.client:opensearch-java:${opensearchVersion}")
     implementation("org.opensearch.plugin:transport-netty4-client:${opensearchVersion}")
     implementation("io.netty:netty-all:${nettyVersion}")
+    implementation("jakarta.inject:jakarta.inject-api:2.0.0")
```

Can we pull the version from `jakartaVersion` i.e. 2.1.1?
```suggestion
    implementation("jakarta.inject:jakarta.inject-api:${jakartaVersion}")
```